framework/src/mesh/MooseMesh.C

```diff
@@ -1980,9 +1978,19 @@ MooseMesh::buildSideList(std::vector<dof_id_type> & el,
                          std::vector<unsigned short int> & sl,
                          std::vector<boundary_id_type> & il)
 {
+  // TODO: Officially deprecate this routine when it is no longer being used by PenetrationLocator.
```

Comment:
Wait, we should be able to deprecate this now. The whole point of the new deprecation system is that it doesn't trigger errors in the TestHarness.

Reply:
OK, wasn't sure about this, I will uncomment that.